src/components/mirror.js

```diff
@@ -48,15 +26,6 @@ AFRAME.registerComponent("mirror", {
       textureWidth: DEFAULT_TEXTURE_WIDTH,
       textureHeight: DEFAULT_TEXTURE_HEIGHT
     });
-    const { onBeforeRender } = reflector;
-
-    reflector.onBeforeRender = function(renderer, scene, camera) {
-      setPlayerHeadVisible(true);
```

Comment:
I think we still need to hook on the onBeforeRender to set the third person layer in render target camera before rendering.

Reply:
Hmm yeah. Ideally we should only need to do this once just to add the THIRD_PERSON_ONLY layer to the Reflector camera https://github.com/MozillaReality/three.js/blob/17375caff569ad48e1bef3e673e33f0bdd5d9bab/examples/js/objects/Reflector.js#L27 but the reference is not exposed so we will either need to modify Three to expose it or do some sort of workaround like this.

Reply:
Not sure about how expensive it's to hook into the rendering hooks vs setting the layers upfront but I like the former better than maintaining a patch.